PWGHF/HFC/TableProducer/correlatorD0Hadrons.cxx

```diff
@@ -375,8 +371,9 @@ struct HfCorrelatorD0Hadrons {
           }
           correlationStatus = true;
         }
-        if (std::abs(track.dcaXY()) >= 1. || std::abs(track.dcaZ()) >= 1.)
-          continue; // Remove secondary tracks
+        if (!track.isGlobalTrackWoDCA()) {
+          continue;
+        }
```

Comment:
This could be directly added in the `Filter` of the tracks, see an example here https://github.com/AliceO2Group/O2Physics/blob/992779b62e6eb916c33bc942fc4de22f3e5278df/DPG/Tasks/AOTTrack/tagAndProbeDmesons.cxx#L241

Reply:
Added.